docs/chainlink-keepers/register-upkeep.md

```diff
@@ -63,6 +63,22 @@ Your Upkeeps will be displayed in your list of Active Upkeeps. You must monitor
 You can dynamically create and manage Upkeeps from within your own dApp. To do this you will need to keep track of the Upkeep ID as your contract will use this to subsequently interact with the Keepers registry. The following example displays a smart contract that can create an Upkeep and determine the Upkeep ID. Note your contract should be Keepers-compatible you will need [ERC-677 LINK](../../link-token-contracts/) to fund the Upkeep. You can also program your Upkeep to check its own balance and fund itself by interacting with the registry.
 
 
+### Prerequisites 
+
+You need to meet these prerequisites to ensure the code snippet will work.
+
+
+* Ensure you use the LINK, registry and registrar addresses for your network.
```

Comment:
nit, maybe "Ensure you use the LINK token contract" instead of simply "LINK" just to be more clear?

Reply:
Agree, will change